src/front/wgsl/keywords.rs

```diff
@@ -0,0 +1,133 @@
+pub const RESERVED: &[&str] = &[
```

Comment:
I think back and front should use the same reserved list. Currently just copy-paste.

Reply:
good point! We need a place to share them